### DISCUSSION TITLE
Can Nvim tree show a list of all buffers?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [nvim-tree/nvim-tree.lua#2766] [Q&A] [answered]
I would like a vertical list of all open buffers to see at a glance. I've managed to get this with Neotree. I'm just wondering if this is possible with Nvim tree or not.

**Comment by alex-courtis** [comment]
You can use the "no buffer" filter. The default mapping is `B`.

See `:help nvim-tree.filters.no_buffer`

**Reply by MoltenMonster** [reply]
What about unnamed buffers?

**Reply by alex-courtis** [reply, marked as answer]
Sorry, only files.

We built a little bit of integration with [aerial](https://github.com/stevearc/aerial.nvim), which may meet your needs: #2173

This reminds me that I need to aerial to my configuration and add a recipe.
